docs/proposals/llm-hyperparameters-tuning-api.md

```diff
@@ -0,0 +1,135 @@
+# LLM Hyperparameters Tuning API Proposal
+
+- [LLM Hyperparameters Tuning API Proposal](#llm-hyperparameters-tuning-api-proposal)
+  * [Links](#links)
+  * [Motivation](#motivation)
+  * [Goal](#goal)
+  * [Design for API](#design-for-api)
+    + [Example](#example)
+  * [Implementation](#implementation)
+  * [Advanced Functionalities](#advanced-functionalities)
+
+## Links
+
+- [katib/issues#2291 (Tuning API in Katib for LLMs)](https://github.com/kubeflow/katib/issues/2291)
+
+## Motivation
+
+LLMs are experiencing rapid growth in popularity and evolution, leading to an increased demand for LLMOps in Kubernetes environment. Hyperparameter tuning can greatly improve model performance but is often labor-intensive for humans. We want to provide an LLM hyperparameters tuning API which automates the process and is user-friendly for data scientists.
+
+## Goal
+
+We aim to develop a high-level API for tuning LLM hyperparameters, which abstracts away infrastructure complexities for data scientists, and facilitates easy hyperparameter tuning by enabling the seamless import of pretrained models and datasets from platforms like HuggingFace and S3, thereby improving accessibility and ease of use.
+
+## Design for API
+
+![Design for API](https://github.com/helenxie-bit/kubeflow/blob/1cdff9f77d68df824bea128e419dc612e172173b/design_api.jpg)
```

Comment:
You can add this diagram, under `proposals/images/design_api.jpg`

Reply:
Thank you for your suggestions! I have modified in this way.